flake.nix

```diff
@@ -37,6 +64,9 @@
             inputs'.holochain.packages.hc-scaffold
           ];
         };
+
+        # TODO: Expose the scaffolding tool CLI as the main package for this crate
+        # packages.default = inputs'.holochain.packages.hc-scaffold;
```

Comment:
Mmm we _could_, but it wouldn't actually be exposing the package coming from the source code of this repository, it would be exposing the one that's coming from the holochain main repository, defined [here](https://github.com/holochain/holochain/blob/392bdfd729fb6ce50f78f9e7f1c757dc392675f4/versions/weekly/flake.nix#L22) and built [here](https://github.com/holochain/holochain/blob/develop/nix/modules/scaffolding.nix). I think the build logic for scaffolding should be declared in this repository, and the main holochain repo should just point to it instead declaring it in the main holochain repository. What do you think @steveej ?